website/addons/dataverse/static/dataverseUserConfig.js

```diff
@@ -123,6 +159,26 @@ function ViewModel(url) {
             }, timeout);
         }
     };
+
+    // Update observables with data from the server
+    $.ajax({
```

Comment:
Note that this will require you to export the ViewModel--we are moving away from the "autobinding" constructors like `DataverseUserConfig`, which are a pain for reuseability and testability.

```js
module.exports = {
    DataverseViewModel: ViewModel,  // preferred
    DataverseUserConfig: DataverseUserConfig  // for backwards-compat
}
```